generic/VarConn.F90

```diff
@@ -147,6 +147,7 @@ logical function checkUnused(this)
       integer                               :: I
       class(Logger), pointer :: lgr
       type(VarConnType), pointer :: conn
+      character(len=ESMF_MAXSTR)            :: from_name, to_name
```

Comment:
Of course part of the functionality of pFlogger is to _avoid_ needing to declare additional variables like this.  Sigh.